Tasks/count.js

```diff
@@ -1,16 +1,13 @@
+'use strict';
+
 // Sum all number values in dict
 
-count = (obj) => {
-  sum = 0;
-  {
+const count = (obj) => {
+  let sum = 0;
+  const values = Object.values(obj);
+  for (const item of values) {
+    if (typeof item === 'number') sum += item;
```

Comment:
Naming hint: `value` or `element`
```suggestion
  for (const value of values) {
    if (typeof value === 'number') sum += value;
```